flutter/macos/Classes/SentryFlutterPlugin.swift

```diff
@@ -0,0 +1,19 @@
+import Cocoa
+import FlutterMacOS
+
+public class SentryFlutterPlugin: NSObject, FlutterPlugin {
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(name: "sentry_flutter", binaryMessenger: registrar.messenger)
+    let instance = SentryFlutterPlugin()
+    registrar.addMethodCallDelegate(instance, channel: channel)
+  }
+
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+    switch call.method {
+    case "getPlatformVersion":
+      result("macOS " + ProcessInfo.processInfo.operatingSystemVersionString)
+    default:
+      result(FlutterMethodNotImplemented)
```

Comment:
this will throw `FlutterMethodNotImplemented` unless we dont call the plugin, another approach is to do like `SentryFlutterWeb`, just return '' to guarantee we don't fail

Reply:
This will not get called at all. There are tests for that.

Reply:
The dummy implementation still has to be removed too.